src/lib/components/controls/MappingCard.svelte

```diff
@@ -23,8 +23,8 @@
 	}
 
 	function handleDelete() {
-		if (confirm('Are you sure you want to delete this mapping?')) {
-			onDelete?.(mapping.uuid);
+		if (confirm('Are you sure you want to delete this mapping?'+ mapping.uuid  + '')) {
```

Comment:
```suggestion
		if (confirm('Are you sure you want to delete this mapping?' + mapping.uuid  + '')) {
```

Reply:
needs s space, also does this even make sense since there may be multiple with the same UUID